docs/guides/verify-steth-on-optimism-upgrade-manual.md

```diff
@@ -0,0 +1,186 @@
+# stETH on Optimism parameters for Mainnet
+
+## Deployment scope
+
+The full list of the contracts in scope is provided in the Lido Multichain section for [Optimism](https://docs.lido.fi/deployed-contracts/#optimism).
+
+## Ethereum part
+
+### TokenRateNotifier
+
+:::info
+A standalone (non-proxy) contract proposed to plug to the protocol as a new [token rebase receiver](/contracts/lido-locator#posttokenrebasereceiver).
+
+Deployed address: [`0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823`](https://etherscan.io/address/0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823)
+:::
+
+```bash
+# Account that is allowed to add or remove pushers.
+TOKEN_RATE_NOTIFIER_OWNER=0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c
```

Comment:
As I understand the document is intended for reading by those who wants to verify already deployed setup. So it is better to be structured not around env variables set upon deploying but around parameters observable in etherscan. Namely, arguments of constructors, initialialize functions and permissions setup.

Reply:
Fair point! Will make another iteration

Reply:
done